staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1105,6 +1105,9 @@ type ManagedFieldsEntry struct {
 	// Time is timestamp of when these fields were set. It should always be empty if Operation is 'Apply'
 	// +optional
 	Time *Time `json:"time,omitempty" protobuf:"bytes,4,opt,name=time"`
+	// FieldsType identifies the format of raw bytes in Fields below.
```

Comment:
"below" is terrible because it won't work in the docs :-)

Reply:
names in the comment should match the json field names.